Firestore/core/src/nanopb/nanopb_util.h

```diff
@@ -104,6 +104,69 @@ T* _Nonnull MakeArray(pb_size_t count) {
   return static_cast<T*>(calloc(count, sizeof(T)));
 }
 
+template <typename T>
```

Comment:
Consider adding tests for some of the newly-added functionality.

Reply:
Tests added for SetRepeatedField